test/parallel/test-net-server-connections-child-null.js

```diff
@@ -20,8 +20,8 @@ if (process.argv[2] === 'child') {
 
     const server = net.createServer();
 
-    // server.connections should start as 0
-    assert.strictEqual(server.connections, 0);
+    // server connections should start as 0
```

Comment:
s/as/at/

Also it might be better to say something like 'server connection count should start at 0'.